idptest/tests/test_google_apps.py

```diff
@@ -7,12 +7,13 @@
 SAML_REQUEST = codex.deflate_and_base64_encode(
     '<?xml version="1.0" encoding="UTF-8"?>'
     '<samlp:AuthnRequest xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" '
+    'xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" '
     'ID="doljiidhacjcjifebimhedigpeejhpifpdmlbjai" Version="2.0" '
     'IssueInstant="2011-10-05T17:49:29Z" '
     'ProtocolBinding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" '
     'ProviderName="google.com" IsPassive="false" '
     'AssertionConsumerServiceURL="https://www.google.com/a/example.com/acs">'
-    '<saml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion">'
+    '<saml:Issuer>'
```

Comment:
This request is generated by the SP right? In which case, can we change it?

I don't understand why we can't have the namespace on the `saml` object anyway, since it's not used anywhere else.